lib/fluent/plugin/exception_detector.rb

```diff
@@ -93,8 +93,15 @@ def self.supported
       rule(:ruby, /^[\t ]+.*?\.rb:\d+:in `/, :ruby)
     ].freeze
 
+    DART_RULES = [
+        rule(:start_state, /^Unhandled exception:$/, :dart_after_exception),
+        rule(:dart_after_exception, /^#\d*\s*.*?\(.*?\)$|^<asynchronous suspension>$/, :dart),
+        rule(:dart_after_exception, /^.*$/, :dart_after_exception),
+        rule(:dart, /^#\d*\s*.*?\(.*?\)$|^<asynchronous suspension>$/, :dart),
+    ].freeze
+
     ALL_RULES = (
-      JAVA_RULES + PYTHON_RULES + PHP_RULES + GO_RULES + RUBY_RULES).freeze
+      JAVA_RULES + PYTHON_RULES + PHP_RULES + GO_RULES + RUBY_RULES + DART_RULES).freeze
```

Comment:
Test, please? You need to add example exceptions to https://github.com/GoogleCloudPlatform/fluent-plugin-detect-exceptions/blob/master/test/plugin/test_exception_detector.rb.

Reply:
I've included two example exceptions.